release/k3s/k3s.go

```diff
@@ -430,24 +436,28 @@ func (r *Release) PushTags(ctx context.Context, tagsCmds []string, ghClient *git
 	if err != nil {
 		return err
 	}
+	var wg sync.WaitGroup
 	for _, tagCmd := range tagsCmds {
-		tag := strings.Split(tagCmd, " ")[3]
-
-		if err := repo.Push(&git.PushOptions{
-			RemoteName: remote,
-			Auth:       gitAuth,
-			Progress:   os.Stdout,
-			RefSpecs: []config.RefSpec{
-				config.RefSpec("+refs/tags/" + tag + ":refs/tags/" + tag),
-			},
-		}); err != nil {
-			if err == git.NoErrAlreadyUpToDate {
-				continue
+		wg.Add(1)
+		tagCmdStr := tagCmd
```

Comment:
Is this assignment necessary?

Reply:
Unsure if the assignment is necessary, although the code does function to the desired effect.